kittens/panel/main.py

```diff
@@ -28,13 +28,37 @@
 --lines
 type=int
 default=1
-The number of lines shown in the panel if horizontal. Ignored for background panels.
+The number of lines shown in the panel. Ignored for background and vertical panels.
 
 
 --columns
 type=int
 default=1
-The number of columns shown in the panel if vertical. Ignored for background panels.
+The number of columns shown in the panel. Ignored for background and horizontal panels.
+
+
+--margin-top
+type=int
+default=0
+Request a given top margin to the compositor.
```

Comment:
Add that the margin only works on Wayland compositor support layer shell. Add to help text for all margin options.